doc/nrf/releases_and_maturity/releases/release-notes-2.8.0.rst

```diff
@@ -0,0 +1,1458 @@
+    * To set ``7`` as a fallback SMS service center address for type approval SIM cards which do not have it set.
+
+* :ref:`lib_at_shell` library:
+
+  * Added the :kconfig:option:`CONFIG_AT_SHELL_UNESCAPE_LF` Kconfig option to enable reception of multiline AT commands.
+  * Updated the :c:func:`at_shell` function to replace ``\n`` with ``<CR><LF>`` if :kconfig:option:`CONFIG_AT_SHELL_UNESCAPE_LF` is enabled.
+
+* :ref:`modem_key_mgmt` library:
+
+  * Updated the :c:func:`modem_key_mgmt_read()` function to return the actual size buffer required to read the certificate if the size provided is too small.
+
+Multiprotocol Service Layer libraries
+-------------------------------------
+
+* Added a 1-wire coexistence implementation that you can enable using the Kconfig option :kconfig:option:`CONFIG_MPSL_CX_1WIRE`.
+* Updated the name of the Kconfig option ``CONFIG_MPSL_CX_THREAD`` to :kconfig:option:`CONFIG_MPSL_CX_3WIRE` to better indicate multiprotocol compatibility.
+* Fixed an issue where the HFXO would be left on after uninitializing MPSL when the RC oscillator was used as the Low Frequency clock source (DRGN-22809).
+* Deprecated the Kconfig option :kconfig:option:`CONFIG_MPSL_CX_BT_1WIRE`.
+
+Libraries for networking
+------------------------
+
+* :ref:`lib_lwm2m_client_utils` library:
+
+  * Updated to use the :ref:`at_parser_readme` library instead of the :ref:`at_cmd_parser_readme` library.
+
+* :ref:`lib_nrf_cloud_rest` library:
+
+  * Added the function :c:func:`nrf_cloud_rest_shadow_transform_request` to request shadow data using a JSONata expression.
+
+* :ref:`lib_nrf_cloud` library:
+
+  * Added:
+
+    * The function :c:func:`nrf_cloud_client_id_runtime_set` to set the device ID string if the :kconfig:option:`CONFIG_NRF_CLOUD_CLIENT_ID_SRC_RUNTIME` Kconfig option is enabled.
+    * The functions :c:func:`nrf_cloud_sec_tag_set` and :c:func:`nrf_cloud_sec_tag_get` to set and get the sec tag used for nRF Cloud credentials.
+    * A new nRF Cloud event type ``NRF_CLOUD_EVT_RX_DATA_DISCON``, which is generated when a device is deleted from nRF Cloud.
+    * The functions :c:func:`nrf_cloud_print_details` and :c:func:`nrf_cloud_print_cloud_details` to log common nRF Cloud connection information in a uniform way.
+    * The :kconfig:option:`CONFIG_NRF_CLOUD_PRINT_DETAILS` Kconfig option to enable the above functions.
+    * The :kconfig:option:`CONFIG_NRF_CLOUD_VERBOSE_DETAILS` Kconfig option to print all details instead of only the device ID.
+    * Experimental support for shadow transform requests over MQTT using the :c:func:`nrf_cloud_shadow_transform_request` function.
+      This functionality is enabled by the :kconfig:option:`CONFIG_NRF_CLOUD_MQTT_SHADOW_TRANSFORMS` Kconfig option.
+    * The :kconfig:option:`CONFIG_NRF_CLOUD_COMBINED_CA_CERT_SIZE_THRESHOLD` and :kconfig:option:`CONFIG_NRF_CLOUD_COAP_CA_CERT_SIZE_THRESHOLD` Kconfig options to compare with the current root CA certificate size.
```

Comment:
The Kconfig option CONFIG_NRF_CLOUD_COMBINED_CA_CERT_SIZE_THRESHOLD is not linking.

Reply:
Is NRF_CLOUD_COAP_CA_CERT_SIZE_THRESHOLD the correct one? @peknis - Could you please confirm this?

Reply:
At least CONFIG_NRF_CLOUD_COAP_CA_CERT_SIZE_THRESHOLD is found in the Kconfig reference, but the ...COMBINED_... is not. Checking this with @plskeggs .

Reply:
@peknis looks like during code review I failed to correct the release notes in my PR that introduced this addition. It should be this:
```suggestion
    * The :kconfig:option:`CONFIG_NRF_CLOUD_AWS_CA_CERT_SIZE_THRESHOLD` and :kconfig:option:`CONFIG_NRF_CLOUD_COAP_CA_CERT_SIZE_THRESHOLD` Kconfig options to compare with the current root CA certificate size.
```